crates/swc/tests/tsc-references/exportAssignmentMergedInterface.1.normal.js

```diff
@@ -8,9 +8,8 @@ define([
 //// [foo_1.ts]
 define([
     "require",
-    "exports",
-    "./foo_0"
-], function(require, exports, _foo_0) {
```

Comment:
This is a bugfix.
foo_0 is not used and removed.

```ts repro
// @module: amd
// @filename: foo_0.ts
interface Foo {
	(a: string): void;
	b: string;
}
interface Foo {
	(a: number): number;
	c: boolean;
	d: {x: number; y: number};
}
export = Foo;

// @filename: foo_1.ts
import foo = require("./foo_0");
var x: foo;
x("test");
x(42);
var y: string = x.b;
if(!!x.c){ }
var z = {x: 1, y: 2};
z = x.d;
```

[Workbench Repro](https://www.typescriptlang.org/dev/bug-workbench/?module=2&ts=5.2.2#code/PTAEAEFsHsBMFcA2BTAXKAhpWAoEEAzASxQDss1QDpoB9ABgDoAXAZxyNOeQCcCMAxslAAxGqADeOAJAAKDOlbMenAOYBKdADdoRWAG4ZAI0XK1hgL4cuvfkNHipchaFLxIR3ptfvPPQ9IC6EY0KBikAbDoEgAe6G4evPqgAJ7xvrwWljjIMQAO0DzMoAC8DtCGeGDgxGQU6NR0AIws7ESQBUVU4mU8yACO8ER9sgBEjMCNDKPqhloYPKBx3RU4MWPcSjOG6wAsAEyzOPOLaaBKKqSqpUuMRoZEBLIAhM8xjALqEqBWJ6AAXjdYugmgAaVLofZZHCAsrvAxAA)